### PR TITLE
Moved NM Services to netmon_libs

### PR DESCRIPTION
**Pull request by alexweltman**
* all angular services WE add are kept in netmon_libs for easy kibana updating
* moved function from kibana controller into it's own service for the same reason (keep as much of our code out of kibana files as possible)

### CODE REVIEW

**Comment by yannick-polius** [comment]
:+1:

File: src/kibana/netmon_libs/custom_modules/save_rule/services/saveRuleManager.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+define(function (require) {
+   
+  var app = require('modules').get('app/dashboard');
```
Comment (john-gress) [inline src/kibana/netmon_libs/custom_modules/save_rule/services/saveRuleManager.js]: Should I give up on consistent indentation in javascript files?

Comment (alexweltman) [inline src/kibana/netmon_libs/custom_modules/save_rule/services/saveRuleManager.js]: it's just the kibana files haha. lemme fix it

**Comment by john-gress** [comment]
👍  But indentation: 😢

**Pull request merged**